src/import/FSHImporter.ts

```diff
@@ -578,6 +590,32 @@ export class FSHImporter extends FSHVisitor {
     });
   }
 
+  visitParamRuleSet(ctx: pc.ParamRuleSetContext): void {
+    const paramRuleSet = new ParamRuleSet(ctx.SEQUENCE().getText())
+      .withLocation(this.extractStartStop(ctx))
+      .withFile(this.currentFile);
+    if (this.paramRuleSets.has(paramRuleSet.name)) {
+      logger.error(`Skipping RuleSet: a RuleSet named ${paramRuleSet.name} already exists.`, {
+        file: this.currentFile,
+        location: this.extractStartStop(ctx)
+      });
+    } else {
+      paramRuleSet.parameters = ctx
+        .PARAMETER_DEF_LIST()
+        .getText()
+        .replace(/(^\()|(\)$)/g, '')
+        .split(',')
+        .map(param => param.trim());
+      paramRuleSet.contents = this.visitParamRuleSetContent(ctx.paramRuleSetContent());
+      this.paramRuleSets.set(paramRuleSet.name, paramRuleSet);
+    }
+  }
+
+  visitParamRuleSetContent(ctx: pc.ParamRuleSetContentContext): string {
+    const result = ctx.start.getInputStream().getText(ctx.start.start, ctx.stop.stop);
+    return result;
```

Comment:
`ctx.getText()` returns the matched tokens, but without the spaces that separated them, since those were piped to the `HIDDEN` channel. Accessing the input stream allows us to get the original contents.

Reply:
Ah right.  I forgot about that.  OK.  Makes sense!